Status crashes with 'java.lang.AssertionError: No System TLS' error upon lauch

# Bug Report

## Problem

Can't start Status (on Android 9). App crashes with 
```
05-26 17:57:18.676 21553 21635 D NetworkSecurityConfig: No Network Security Config specified, using platform default
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: Failed to create NativeModule "UIManager"
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: java.lang.AssertionError: No System TLS
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: 	at okhttp3.internal.Util.assertionError(Util.java:504)
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: 	at okhttp3.OkHttpClient.newSslSocketFactory(OkHttpClient.java:296)
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: 	at okhttp3.OkHttpClient.<init>(OkHttpClient.java:258)
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: 	at okhttp3.OkHttpClient$Builder.build(OkHttpClient.java:1040)
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: 	at com.reactnativecommunity.webview.RNCWebViewManager.<init>(RNCWebViewManager.java:182)
05-26 17:57:18.684 21553 21635 E unknown:NativeModuleInitError: 	at com.reactnativecommunity.webview.RNCWebViewPackage.createViewManagers(RNCWebViewPackage.kt:13)
05-26 17:57:
...
```

It's all fine on Android 8.1 but on Android 9 I get crashes all the time on both Nightly Develop and PR builds

E2E tests which are running on Android 10, sometimes may may crash the session with the same error (but it's 1 of 20 times)
#### Expected behavior

Status app starts
#### Actual behavior

App crash when launch

![323](https://user-images.githubusercontent.com/8749671/82917159-b5510100-9f7b-11ea-988b-3b568fb70673.gif)

### Reproduction

- Install Status build https://status-im.ams3.digitaloceanspaces.com/StatusIm-200526-025900-2f65ce-nightly-universal.apk on Android 9
- Open Status

### Additional Information

[comment]: # (Please do your best to fill this out.)
- Status version: https://status-im.ams3.digitaloceanspaces.com/StatusIm-200526-025900-2f65ce-nightly-universal.apk
- Operating System: Android 9, Android 10

#### Logs

Full system logcat:  
[appcrash.txt](https://github.com/status-im/status-react/files/4683080/appcrash.txt)

--- a/src/mocks/js_dependencies.cljs
+++ b/src/mocks/js_dependencies.cljs
@@ -1,5 +1,4 @@
 (ns mocks.js-dependencies
-  (:require-macros [status-im.utils.slurp :refer [slurp]])
   (:require [status-im.default-fleet :refer (default-fleets)]))
 
 (def action-button          #js {:default #js {:Item #js {}}})
@@ -192,20 +191,4 @@
     "react-native-mail" react-native-mail
     "react-native-image-resizer" image-resizer
     "./fleets.js" default-fleets
-    "../translations/ar.json" (js/JSON.parse (slurp "./translations/ar.json"))
-    "../translations/de.json" (js/JSON.parse (slurp "./translations/de.json"))
-    "../translations/en.json" (js/JSON.parse (slurp "./translations/en.json"))
-    "../translations/es.json" (js/JSON.parse (slurp "./translations/es.json"))
-    "../translations/es_419.json" (js/JSON.parse (slurp "./translations/es_419.json"))
-    "../translations/fil.json" (js/JSON.parse (slurp "./translations/fil.json"))
-    "../translations/fr.json" (js/JSON.parse (slurp "./translations/fr.json"))
-    "../translations/id.json" (js/JSON.parse (slurp "./translations/id.json"))
-    "../translations/it.json" (js/JSON.parse (slurp "./translations/it.json"))
-    "../translations/ko.json" (js/JSON.parse (slurp "./translations/ko.json"))
-    "../translations/pt_br.json" (js/JSON.parse (slurp "./translations/pt_br.json"))
-    "../translations/ru.json" (js/JSON.parse (slurp "./translations/ru.json"))
-    "../translations/tr.json" (js/JSON.parse (slurp "./translations/tr.json"))
-    "../translations/zh.json" (js/JSON.parse (slurp "./translations/zh.json"))
-    "../translations/zh_hant.json" (js/JSON.parse (slurp "./translations/zh_hant.json"))
-    "../translations/zh_TW.json" (js/JSON.parse (slurp "./translations/zh_TW.json"))
     nil))

--- a/src/status_im/i18n.clj
+++ b/src/status_im/i18n.clj
@@ -1,0 +1,21 @@
+(ns status-im.i18n
+  (:require [clojure.string :as string]))
+
+(defn read-file [f]
+  (when (.isFile f)
+    (let [locale  (-> f
+                      .getName
+                      (string/split #"\.")
+                      first)
+          content (slurp f)]
+      [(keyword locale) content])))
+
+(defn read-translations []
+  (->>
+   (java.io.File. "translations")
+   file-seq
+   (keep read-file)
+   vec))
+
+(defmacro translations [languages]
+  (select-keys (into {} (read-translations)) languages))

--- a/src/status_im/i18n_resources.cljs
+++ b/src/status_im/i18n_resources.cljs
@@ -1,66 +1,21 @@
 (ns status-im.i18n-resources
-  (:require [clojure.string :as string]
-            ["i18n-js" :as i18n-js]
+  (:require-macros [status-im.i18n :as i18n])
+  (:require [status-im.utils.types :as types]
+            [clojure.string :as string]
             ["react-native-languages" :default react-native-languages]))
 
 (def default-device-language
   (keyword (.-language react-native-languages)))
 
-;; Update js_dependencies.cljs
-(def languages [:ar :de :en :es :es_419 :fil :fr :id :in :it :ko :pt_BR :ru :tr :zh :zh_Hant :zh_TW])
-
-(defonce loaded-languages
-  (atom
-   (conj #{:en} default-device-language)))
-
-(def translations
-  {:ar         (js/require "../translations/ar.json")
-   :de         (js/require "../translations/de.json")
-   :en         (js/require "../translations/en.json")
-   :es         (js/require "../translations/es.json")
-   :es_419     (js/require "../translations/es_419.json")
-   :fil        (js/require "../translations/fil.json")
-   :fr         (js/require "../translations/fr.json")
-   :id         (js/require "../translations/id.json")
-   :in         (js/require "../translations/id.json")
-   :it         (js/require "../translations/it.json")
-   :ko         (js/require "../translations/ko.json")
-   :pt_BR      (js/require "../translations/pt_br.json")
-   :ru         (js/require "../translations/ru.json")
-   :tr         (js/require "../translations/tr.json")
-   :zh         (js/require "../translations/zh.json")
-   :zh_Hant    (js/require "../translations/zh_hant.json")
-   :zh_TW      (js/require "../translations/zh_TW.json")})
-
-(defn valid-language [lang]
-  (if (contains? translations lang)
-    (keyword lang)
-    (let [parts (string/split (name lang) #"[\-\_]")
-          short-lang (keyword (str (first parts) "_" (second parts)))
-          shortest-lang (keyword (first parts))]
-      (if (and (> (count parts) 2) (contains? translations short-lang))
-        short-lang
-        (when (contains? translations shortest-lang)
-          shortest-lang)))))
-
-(defn require-translation [lang-key]
-  (when-let [lang (valid-language (keyword lang-key))]
-    (get translations lang)))
-
 ;; translations
 (def translations-by-locale
-  (cond->
-   {:en (require-translation :en)}
-    (not= :en default-device-language)
-    (assoc default-device-language
-           (require-translation (-> (name default-device-language)
-                                    (string/replace "-" "_")
-                                    keyword)))))
+  (->> (i18n/translations [:ar :en :es :es_419 :fil :fr :it :ko :ru :in :id :tr :zh :zh_Hant :zh_TW])
+       (map (fn [[k t]]
+              (let [k' (-> (name k)
+                           (string/replace "_" "-")
+                           keyword)]
+                [k' (types/json->clj t)])))
+       (into {})))
 
-(defn load-language [lang]
-  (when-let [lang-key (valid-language (keyword lang))]
-    (when-not (contains? @loaded-languages lang-key)
-      (aset (.-translations i18n-js)
-            lang
-            (require-translation lang-key))
-      (swap! loaded-languages conj lang-key))))
+;; API compatibility
+(defn load-language [_])

--- a/src/status_im/i18n_test.cljs
+++ b/src/status_im/i18n_test.cljs
@@ -7,15 +7,14 @@
             [clojure.string :as string]))
 
 ;; english as source of truth
-(def labels (set (keys (js->clj (:en i18n-resources/translations-by-locale)
-                                :keywordize-keys true))))
+(def labels (set (keys (:en i18n-resources/translations-by-locale))))
 
 (spec/def ::label labels)
 (spec/def ::labels (spec/coll-of ::label :kind set? :into #{}))
 
 (defn labels-for-all-locales []
   (->> i18n-resources/translations-by-locale
-       (mapcat #(-> % val (js->clj :keywordize-keys true) keys))
+       (mapcat #(-> % val keys))
        set))
 
 ;; checkpoints
@@ -1031,7 +1030,7 @@
 (spec/def ::locales (spec/coll-of ::locale :kind set? :into #{}))
 
 (defn locale->labels [locale]
-  (-> i18n-resources/translations-by-locale (get locale)  (js->clj :keywordize-keys true) keys set))
+  (-> i18n-resources/translations-by-locale (get locale) keys set))
 
 (defn locale->checkpoint [locale]
   (let [locale-labels (locale->labels locale)

--- a/src/status_im/node/core.cljs
+++ b/src/status_im/node/core.cljs
@@ -7,7 +7,8 @@
             [status-im.utils.config :as config]
             [status-im.utils.fx :as fx]
             [status-im.utils.platform :as utils.platform]
-            [status-im.utils.types :as types]))
+            [status-im.utils.types :as types])
+  (:require-macros [status-im.utils.slurp :refer [slurp]]))
 
 (defn- add-custom-bootnodes [config network all-bootnodes]
   (let [bootnodes (as-> all-bootnodes $
@@ -69,8 +70,10 @@
   [limit nodes]
   (take limit (shuffle nodes)))
 
+(def default-fleets (slurp "resources/config/fleets.json"))
+
 (defn fleets [{:keys [custom-fleets]}]
-  (as-> [(js/require "./fleets.js")] $
+  (as-> [default-fleets] $
     (mapv #(:fleets (types/json->clj %)) $)
     (conj $ custom-fleets)
     (reduce merge $)))

